meta/style.pm

```diff
@@ -1253,6 +1285,8 @@ sub CheckHeadersStyle
             next if $line =~ /^ {4}[{}]/;           # start or end of union
             next if $line =~ /^ {4}(u?int)/;        # union entries
             next if $line =~ /^ {4}(char|bool)/;    # union entries
+            next if $line =~ /^ {4}float/;          # union entries
```

Comment:
if double is used, it should be used everywehre instead of float, for better precision